falcon/response.py

```diff
@@ -434,3 +434,10 @@ def _wsgi_headers(self, media_type=None):
             return headers.items()
 
         return list(headers.items())  # pragma: no cover
+
+    def redirect(self, uri, permanent=False):
+        if permanent:
+            self.status = "301 Moved Permanently"
+        else:
+            self.status = "303 See Other"
+        self.append_header("Location", uri)
```

Comment:
I think we could use `self.set_header` instead, since we don't expect multiple Location headers.